datasets/covid_tweets_japanese/README.md

```diff
@@ -83,7 +83,7 @@ CSV file with the 1st column is Twitter ID and the 2nd column is assessment opti
 
 ### Data Splits
 
-[More Information Needed]
+No information about data splits is provided for now.
```

Comment:
It means that there's no predefined splits right ?

Also not sure about using "for now", does that mean that data will be split at one point ?

Reply:
@lhoestq 
Thank you for your quick checking of this PR!
I'm sorry for my lack of words.

I used "for now" because the author of the dataset says 「このデータに関係する論文は**まだ**公開されていません．」("a related paper of this dataset has not **yet** published.") on the website.

So, what I should have said is that "A related paper of this dataset has not been published, and it seems that the author of the dataset has the willingness to publish a paper (whether the information of splitting is included is not sure). Hence, for now, the information about data splits is not provided."

Does the library have a unified view on how to write information in such a case?

Reply:
I think you can just write what you just said ^^
Maybe something like
```
No articles have been published for this dataset, and it appears that the author of the dataset is willing to publish an article (it is not certain that the splitting information will be included). Therefore, at this time, information on data splits is not provided.
```
What do you think ?

Reply:
Thanks, I'll update the explanation right now!

Reply:
@lhoestq 
I've updated the PR by using the text you've kindly summarized. It has passed all the checks. 
Please check it again when you have time.